bindings/cs/rl.net.cli.test/TempFileDisposable.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.IO;
+
+namespace Rl.Net.Cli.Test
+{
+    internal sealed class TempFileDisposable : IDisposable
+    {
+        public TempFileDisposable()
+        {
+            this.Path = System.IO.Path.GetTempFileName();
+        }
+
+        public string Path
+        {
+            get;
+            private set;
+        }
+
+        public void Dispose()
+        {
+            try
+            {
+                if (File.Exists(this.Path))
+                {
+                    File.Delete(this.Path);
+                }
+
+                if (Directory.Exists(this.Path))
```

Comment:
It should not be able to, since `.Path` is the result of Path.GetTempFileName(). The idea is to ensure it deletes directories created in this manner, since `File.Exists(path)` returns false if `path` points to a directory.